src/Controls/src/Core/Editor/Editor.iOS.cs

```diff
@@ -9,6 +9,9 @@ public static void MapText(EditorHandler handler, Editor editor) =>
 		public static void MapText(IEditorHandler handler, Editor editor)
 		{
 			Platform.TextExtensions.UpdateText(handler.PlatformView, editor);
+
+			// Any text changes in the editor field require recalculating the CharacterSpacing by regenerating the attributed string to properly apply the spacing and override the current text formatting.
+			handler?.UpdateValue(nameof(CharacterSpacing));
```

Comment:
The fix looks good but, trying to understand why is necessary because is something we already are doing.

In the EditorHandler, updating the Text property we are invoking the MapFormatting method: https://github.com/dotnet/maui/blob/f269ef3de701043910c941bbbfcbdb7422cc245c/src/Core/src/Handlers/Editor/EditorHandler.iOS.cs#L85 

Where update the CharacterSpacing after set the text: https://github.com/dotnet/maui/blob/f269ef3de701043910c941bbbfcbdb7422cc245c/src/Core/src/Handlers/Editor/EditorHandler.iOS.cs#L138

Reply:
@jsuarezruiz,

Since, we are remapping in editor text Property, which directly calls Editor.iOS MapText on text changes instead of MapText in EntryHandler.iOS. So, the MapText function in EntryHandler.iOS is not invoked whenever text changes.

![image](https://github.com/user-attachments/assets/28c6bf7c-20a2-437c-80ed-6f6aefa38acd)

Additionally, we’re already handling MaxLength in [UpdateText](https://github.com/dotnet/maui/blob/ad6f752f70ce97846fc3e0e4becd0815153fcb1f/src/Core/src/Core/Extensions/ITextInputExtensions.cs#L9), so there’s no need to update UpdateMaxLength through MapFormatting, as we ensure the length limitation within UpdateText itself. By doing this, the MapText method remains efficient and prevents unnecessary reprocessing, focusing solely on applying changes relevant to CharacterSpacing.

https://github.com/dotnet/maui/blob/ad6f752f70ce97846fc3e0e4becd0815153fcb1f/src/Core/src/Core/Extensions/ITextInputExtensions.cs#L18-L19